### PR TITLE
 Add 'is_ssl_enabled' for bug 1851610 to check if SSL module is enabled

### PR DESCRIPTION
**Pull request by mullumaus**
Add 'is_ssl_enable' for bug 1851610 to check if SSL module is enabled or not

The gerrit review link:  https://review.opendev.org/c/openstack/charm-neutron-api/+/766883

### CODE REVIEW

File: charmhelpers/contrib/hardening/audits/apache.py [hunk 1]
```diff
@@ -98,3 +98,8 @@ def _disable_module(module):
     def _restart_apache():
         """Restarts the apache process"""
         subprocess.check_output(['service', 'apache2', 'restart'])
+
+    @staticmethod
+    def is_ssl_enable():
```
Comment (wolsen) [inline charmhelpers/contrib/hardening/audits/apache.py:103]: The method should be named `is_ssl_enabled` vs `is_ssl_enable`.

**Review by wolsen — CHANGES_REQUESTED** [review]
In addition to the inline comment, can we please add a unit test?

File: charmhelpers/contrib/hardening/audits/apache.py (hunk 1 above)
Comment (mullumaus) [reply charmhelpers/contrib/hardening/audits/apache.py]: Thanks Billy, I removed the method to 'is_ssl_enabled' and added a unit test for it in tests/contrib/hardening/audits/test_apache_audits.py

**Review by ajkavanagh — APPROVED** [review]
LGTM - just need @wolsen to take another look.

**Review by wolsen — APPROVED** [review]

**Pull request merged**